components/kyma-environment-broker/internal/httputil/client.go

```diff
@@ -14,3 +15,17 @@ func NewClient(timeoutSec time.Duration, skipCertVerification bool) *http.Client
 		Timeout:   timeoutSec * time.Second,
 	}
 }
+
+func NewRenegotiationTLSClient(timeoutSec time.Duration, skipCertVerification bool) *http.Client {
+	transport := &http.Transport{
```

Comment:
you should init transport struct in th same way as above:
```
transport := http.DefaultTransport.(*http.Transport).Clone()
```

that gives you defaults for some fields, and then you can adjust rest of them